e2e/common/createCaseHelper.ts

```diff
@@ -13,23 +12,17 @@ async function createDaCitizenCourtNavCase(
   withNotice: boolean,
   withDoc: boolean,
 ): Promise<string> {
-  const apiContextDaCreateCase: APIRequestContext = await request.newContext();
-  const tokenDaCreateCase = await getAccessToken(
-    "daCourtNavCreateCase",
-    apiContextDaCreateCase,
-  );
-  if (!tokenDaCreateCase) {
-    throw new Error("Setup failed: Unable to get bearer token.");
-  }
-  process.env.COURTNAV_CREATE_CASE_BEARER_TOKEN = tokenDaCreateCase;
+  const apiContextDaCreateCase2: APIRequestContext = await request.newContext();
+  const tokenDaCreateCase = process.env
+    .COURTNAV_CREATE_CASE_BEARER_TOKEN as string;
```

Comment:
Does COURTNAV_CREATE_CASE_BEARER_TOKEN need to be defined in example.env?

Reply:
doesn't need to be defined but is being stored in a env var for use across the tests